### PR TITLE
Transparent fog

### PR DESCRIPTION
**Pull request by Lwmte**
Fixes an issue with additive / subtractive / etc surfaces having incorrect fog blending.

### CODE REVIEW

**Comment by goldlife2904** [comment]
Had test it - and it works for me.
Test fog with transparent objects and also test Flame Emitters, Flares, Grenades

File: TombEngine/Shaders/DX11_Sprites.fx [hunk 1]
```diff
@@ -15,7 +15,7 @@ struct PixelShaderInput
 	float3 Normal: NORMAL;
 	float2 UV: TEXCOORD1;
 	float4 Color: COLOR;
-	float Fog : FOG;
+	float  Fog : FOG;
```
Comment (Raildex) [inline TombEngine/Shaders/DX11_Sprites.fx:18]: one whitespace too much

Comment (Raildex) [inline TombEngine/Shaders/DX11_Sprites.fx:18]: One white space too much

**Review by Raildex — APPROVED** [review]

File: TombEngine/Shaders/DX11_Sprites.fx (hunk 1 above)
Comment (Lwmte) [reply TombEngine/Shaders/DX11_Sprites.fx:18]: It is intentional to beautify struct alignment :)

**Pull request merged**